python/lsst/sconsUtils/scripts.py

```diff
@@ -168,13 +168,17 @@ def finish(defaultTargets=DEFAULT_TARGETS,
         if "tests" in [str(t) for t in BUILD_TARGETS]:
             testsDir = pipes.quote(os.path.join(os.getcwd(), "tests", ".tests"))
             checkTestStatus_command = state.env.Command('checkTestStatus', [], """
-                @ if [ -d %s ]; then \
-                      nfail=`find %s -name \*.failed | wc -l | sed -e 's/ //g'`; \
+                @ if [ -d {0} ]; then \
+                      nfail=`find {0} -name \*.failed | wc -l | sed -e 's/ //g'`; \
                       if [ $$nfail -gt 0 ]; then \
+                          echo "Failed test output:" >&2; \
+                          find {0} -name \*.failed -exec cat {{}} \; >&2; \
```

Comment:
This was a comprehension failure on my part -- I was thrown off by the escape line endings and didn't catch that the escaped `;` was part of the `-exec` arg.  @timj and I just discussed this and I think it's good as-is.